src/structures/listening_session.ts

```diff
@@ -156,11 +154,11 @@ export default class ListeningSession extends Session {
         }
 
         const round = this.round;
-        const guildID = interaction.guildID;
+        const guildID = interaction.guildID as string;
         if (interaction.data.custom_id === "bookmark") {
             this.handleBookmarkInteraction(interaction);
```

Comment:
Do we need to await this?

Reply:
Don't know but not within the scope of this PR

Reply:
This PR makes `handleBookmarkInteraction` async so we might need to await where appropriate